docs/main/troubleshooting/troubleshooting-replicated-storage.md

```diff
@@ -318,57 +285,57 @@ Running `kubectl get dsp -n mayastor` could result in the error due to the `v1al
 
 When creating a Disk Pool with `kubectl create -f dsp.yaml`, you might encounter an error related to `v1alpha1` CR definitions. To resolve this, ensure your CR definition is updated to `v1beta1` in the YAML file (for example, `apiVersion: openebs.io/v1beta1`).
 
-{% hint style="note" %}
+:::note
 You can validate the schema changes by executing `kubectl get crd diskpools.openebs.io`.
-{% endhint %}
+:::
 
 # Known Limitations
 
 ## Volume and Pool Capacity Expansion
 
-Once provisioned, neither Mayastor Disk Pools nor Mayastor Volumes can be re-sized. A Mayastor Pool can have only a single block device as a member. Mayastor Volumes are exclusively thick-provisioned.
+Once provisioned, neither Replicated Storage Disk Pools nor Replicated Storage Volumes can be re-sized. A Replicated Storage Pool can have only a single block device as a member. Replicated Storage Volumes are exclusively thick-provisioned.
 
 ## Snapshots and Clones
 
-Mayastor has no snapshot or cloning capabilities.
+Replicated Storage has no snapshot or cloning capabilities.
```

Comment:
Change this to "Replicated Storage currently supports provisioning snapshots and clones on volumes with only one replica."